apps/sasquatch/src/androidTest/java/com/microsoft/appcenter/sasquatch/activities/CrashesTest.java

```diff
@@ -184,9 +184,7 @@ private void crashTest(@StringRes int titleId) throws InterruptedException {
                 new Date().getTime() - errorReport.getAppErrorTime().getTime(),
                 lessThan(10000L));
         assertNotNull(errorReport.getDevice());
-        assertEquals(failureHandler.uncaughtException.getClass(), errorReport.getThrowable().getClass());
-        assertEquals(failureHandler.uncaughtException.getMessage(), errorReport.getThrowable().getMessage());
-        assertArrayEquals(failureHandler.uncaughtException.getStackTrace(), errorReport.getThrowable().getStackTrace());
+        assertNotNull(errorReport.getStackTrace());
```

Comment:
Since this is an `androidTest`, we can actually make a comparison equivalent to the deleted lines.

Do something like comparing `Log.getStackTraceString` of the expected object `failureHandler.uncaughtException` and compare to the actual string.